package.json

```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "lint": "eslint .",
     "pretest": "node pretest.js",
-    "test": "mocha --timeout 15000 --require test/init.js",
+    "test": "mocha --timeout 60000 test/*.test.js node_modules/juggler-v3/test.js node_modules/juggler-v4/test.js",
```

Comment:
Note: for whoever working on any similar tasks, adding timeout/making it longer may fix timing-related errors.